### PR TITLE
41 subtitle alternate

### PR DESCRIPTION
**Pull request by fadeomar**
**Issue**: #41 
## What did I do?
create  subtitle alternate that exists in the earwig case study page

## How to test?
```yarn storybook```

### CODE REVIEW

File: src/components/SubtitleAlternate/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { CapsHeadingMedium, MP } from '../elements/typography/index'
+
+const AlternateContainer = styled.div`
+  box-sizing: border-box;
+  margin-top: 30px;
+  position: relative;
+  padding: 50px 90px;
```
[the remark this answers is not in the document]
Comment (Amoodaa) [reply src/components/SubtitleAlternate/index.js]: Maybe i suck at css dunno

[the remark this answers is not in the document]
Comment (fadeomar) [reply src/components/SubtitleAlternate/index.js]: yeah I can see that

File: src/components/SubtitleAlternate/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { CapsHeadingMedium, MP } from '../elements/typography/index'
+
+const AlternateContainer = styled.div`
+  box-sizing: border-box;
+  margin-top: 30px;
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply src/components/SubtitleAlternate/index.js]: explain

File: src/components/SubtitleAlternate/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  background: rgba(247, 247, 247, 0.7);
+`
+
+const PragraphContainer = styled.div`
+  width: 90%;
+`
+const Accent = styled.span`
+  width: 4px;
+  transform: rotate(180deg);
+  height: 42px;
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply src/components/SubtitleAlternate/index.js]: explain

File: src/components/SubtitleAlternate/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  position: relative;
+  padding: 50px 90px;
+  background: rgba(247, 247, 247, 0.7);
+`
+
+const PragraphContainer = styled.div`
+  width: 90%;
+`
+const Accent = styled.span`
+  width: 4px;
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply src/components/SubtitleAlternate/index.js]: explain

File: src/components/SubtitleAlternate/index.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  transform: rotate(180deg);
+  height: 42px;
+  background: #e66767;
+  display: inline-block;
+  position: absolute;
+  top: -21px;
+  left: 60px;
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply src/components/SubtitleAlternate/index.js]: explain

File: src/components/SubtitleAlternate/index.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { CapsHeadingMedium, MP } from '../elements/typography/index'
```
Comment (Amoodaa) [inline src/components/SubtitleAlternate/index.js]: 'index' is automatically imported when u refer to the folder

File: src/components/SubtitleAlternate/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Amoodaa) [reply src/components/SubtitleAlternate/index.js]: change all the px to rem

File: src/components/SubtitleAlternate/index.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { ExtraSmall as Accent } from '../elements/Accent'
+
+import { CapsHeadingMedium, MP } from '../elements/typography'
+
+const AlternateContainer = styled.div`
+  box-sizing: border-box;
```
Comment (yosefanajjar) [inline src/components/SubtitleAlternate/index.js]: remove this cause it's not needed anymore

File: src/components/SubtitleAlternate/index.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+  margin-top: 30px;
```
Comment (yosefanajjar) [inline src/components/SubtitleAlternate/index.js]: we decided to use rems for padding and margins

File: src/components/SubtitleAlternate/SubtitleAlternate.stories.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import SubtitleAlternate from './index'
+
+export default {
+  title: 'subtitle alternate',
```
Comment (Amoodaa) [inline src/components/SubtitleAlternate/SubtitleAlternate.stories.js]: can you move it to `components`

**Pull request merged**